tests/2014-04-21-HPC-GPU_Meetup/slide_13.cpp

```diff
@@ -17,11 +17,16 @@ int main() {
         sycl::buffer<int> inputABuffer(inputA.data(), inputA.size());
         sycl::buffer<int> inputBBuffer(inputB.data(), inputB.size());
         sycl::buffer<int> outputBuffer(output.data(), output.size());
+        sycl::queue *myQueue;
+        try {
+            sycl::context myContext { sycl::gpu_selector { }};
+            myQueue = new sycl::queue(myContext, sycl::gpu_selector { });
+        }
+        catch (cl::sycl::runtime_error &) {
+            myQueue = new sycl::queue();
+        }
```

Comment:
It could be simpler and without pointer, since by default a queue is the host queue and a queue has already a reference semantics.
I think it could be simplified to
```
sycl::queue myQueue;
try {
  sycl::context myContext { sycl::gpu_selector { } };
  myQueue = sycl::queue { myContext, sycl::gpu_selector { } };
}
catch (cl::sycl::runtime_error &) {
  // Do nothing, there is already the host queue inside myQueue
}
```